### PR TITLE
Use workflow future to determine running status

### LINKED ISSUE #913: Possible bug in workflow runner
Multiple users report following error when running workflow
```
Traceback (most recent call last):
  File "/prog/res/komodo/2020.02.01-py27/root/lib/python2.7/site-packages/ert_gui/tools/workflows/run_workflow_widget.py", line 141, in workflowFinishedWithFail
    error = self._workflow_runner.workflowError()
  File "/prog/res/komodo/2020.02.01-py27/root/lib/python2.7/site-packages/res/job_queue/workflow_runner.py", line 78, in workflowError
    for error_line in error:
TypeError: 'NoneType' object is not iterable
```

Conversation found here: https://equinor.slack.com/archives/CDPJULWR4/p1582812077085900

### PR DESCRIPTION
**Pull request by jondequinor**
**Issue**
Resolves #913 

**Approach**
Use future status in `isRunning`

### CODE REVIEW

**Comment by markusdregi** [comment]
Could you please elaborate on why it helps to check the status of the job as well? :)

**Comment by jondequinor** [comment]
> Could you please elaborate on why it helps to check the status of the job as well? :)

Added comment https://github.com/equinor/libres/blob/include-future-in-running-status/python/res/job_queue/workflow_runner.py#L46

File: python/res/job_queue/workflow_runner.py [hunk 1]
```diff
@@ -41,7 +41,14 @@ def __runWorkflow(self):
 
     def isRunning(self):
         """ @rtype: bool """
-        return self.__workflow.isRunning()
+        workflow_running = self.__workflow.isRunning()
```
Comment (markusdregi) [inline python/res/job_queue/workflow_runner.py]: Would if be easier to read as:

```python
if self.__workflow.isRunning():
    return True
return self._workflow_job is not None and not self._workflow_job.done()
```

or something similar?

Comment (jondequinor) [reply python/res/job_queue/workflow_runner.py]: Indeed! There are many ways to skin this beast.

**Review by markusdregi — APPROVED** [review]
🚢

**Pull request merged**